packages/infra/lib/hl7-notification-stack/network.ts

```diff
@@ -19,13 +20,61 @@ export class NetworkStack extends cdk.NestedStack {
   constructor(scope: Construct, id: string, props: NetworkStackProps) {
     super(scope, id, props);
 
-    const vpc = new ec2.Vpc(this, "Vpc", {
-      maxAzs: NUM_AZS,
-      natGateways: NUM_AZS,
```

Comment:
Why not creating the VPC in the NetworkStack anymore? Could have this nested stack be the first one in the main stack's constructor, no?

Reply:
That's what I had before, but the cdk deploys for ECS / Fargate are insanely slow. I put them first so that I could roll forward/ roll back more easily without having to wait an extra 10 minutes for ECS to spin up + down.